es-space-mgmt/main/default/lwc/customerDetailForm/__tests__/customerDetailForm.test.js

```diff
@@ -143,4 +143,37 @@ describe('c-customer-detail-form', () => {
             );
         });
     });
+
+    it('is accessible when customer detail fields returned', () => {
```

Comment:
@albarivas Is there a reason why we duplicate tests within a test suite? That feels like not needed overhead to me.

I acknowledge that the dedicated test message is nice, but at the end does it matter if a test fails b/c an element is null, an event is not fired, or if a components UI state has an accessibility issue? It would be an error nonetheless.

Reply:
I've decided to write dedicated tests for accessibility, as 1) what these tests check is different and concrete enough to consider them "testing one thing", which is one of the principles of unit testing 2) accessibility tests must test all the possible UI configurations, which are not necessary reflected in other tests and 3) because examining the results is much easier this way.